Scripts/Prerequisite.ps1

```diff
@@ -0,0 +1,96 @@
+param(
+    [Parameter(Mandatory = $true)]
+    [string]$subscriptionId,
+    [Parameter(Mandatory = $true)]
+    [string]$resourceGroupName,
+    [Parameter(Mandatory = $true)]
+    [string]$keyVaultName,
+    [Parameter(Mandatory = $true)]
+    [string]$location,
+    [Parameter(Mandatory = $true)]
+    [string]$certName,
+    [Parameter(Mandatory = $true)]
+    [string]$keySecretForServicePrincipal,
+    [Parameter(Mandatory = $true)]
+    [string]$servicePrincipalName,
+    [Parameter(Mandatory = $true)]
+    [string]$tenantId,
+    [Parameter(Mandatory = $true)]
+    [string]$serviceConnectionName,
+    [Parameter(Mandatory = $true)]
+    [string]$organizationName,
+    [Parameter(Mandatory = $true)]
+    [string]$projectName
+)
+
+try {
+
+  $isLoggedIn = az account show 
+  if ($null -eq $isLoggedIn) {
+      Write-Host "logging In..." -ForegroundColor White
+      az login --use-device-code
```

Comment:
please avoid using --use-device-code, it's old wat authentication based on device code. just simply use az login ...it redirects to browser authentication by default.